apps/services/personal-representative/test/setup.ts

```diff
@@ -0,0 +1,76 @@
+import { testServer, TestServerOptions } from '@island.is/infra-nest-server'
+import { getConnectionToken } from '@nestjs/sequelize'
+import { Sequelize } from 'sequelize-typescript'
+import { AppModule } from '../src/app/app.module'
+import type { INestApplication, Type } from '@nestjs/common'
+import { logger } from '@island.is/logging'
+
+export let app: INestApplication
+let sequelize: Sequelize
+
+export const truncate = async () => {
+  if (!sequelize) {
+    return
+  }
+
+  await Promise.all(
+    Object.values(sequelize.models).map((model) => {
+      if (model.tableName.toLowerCase() === 'sequelize') {
+        return null
+      }
+
+      return model.destroy({
+        where: {},
+        cascade: true,
+        truncate: true,
+        force: true,
+      })
+    }),
+  )
+}
+
+// needed for generic error validation
+expect.extend({
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  anyOf(value: any, classTypes: any[]) {
+    const types = classTypes.map((type) => type.name).join(', ')
+    const message = `expected to be any of type: ${types}`
+    for (let i = 0; i < classTypes.length; i++) {
+      if (value.constructor === classTypes[i]) {
+        return {
+          pass: true,
+          message: () => message,
+        }
+      }
+    }
+
+    return {
+      pass: false,
+      message: () => message,
+    }
+  },
+})
+
+export const setup = async (options?: Partial<TestServerOptions>) => {
```

Comment:
Checkout `@island.is/testing/nest` ... it reduces the need for custom setup in each project.